README.md

```diff
@@ -296,6 +296,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Dictionaries
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Google Dictionary API](https://dictionaryapi.dev) | Word definitions, examples, pronunciations, synonyms, and others | No | Yes | Yes |
```

Comment:
Can you change the title `Google Dictionary API` to `Google Dictionary`, please?

```suggestion
| [Google Dictionary](https://dictionaryapi.dev) | Word definitions, examples, pronunciations, synonyms, and others | No | Yes | Yes |
```

Title should not end with "... API". Every entry is an API here! 😅